client/src/components/App.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from "react";
-
+import Header from "./Header/Header"
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <p>Hello Winter of Code</p>
+        <Header></Header>
```

Comment:
Use self closing way for empty elements.
```suggestion
        <Header />
```

Reply:
I'll do that👍🏻